styles/styles.css

```diff
@@ -0,0 +1,495 @@
+  left: 10%;
+  top: 20%;
+  display: flex;
+  /*does this need to say display: flex?*/
+  flex-direction: column;
+  color: #81B642;
+  font-family: Merriweather, Times, serif;
+  font-style: italic;
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.filler {
+  width: 0%;
+}
+
+.toplogo {
+  max-width: 100%;
+  height: auto;
+}
+
+.cart {
+  position: absolute;
+  right: 10%;
+  top: 15%;
+  display: flex;
+  flex-direction: column;
+  justify-content: right;  
+}
+
+.top-cart {
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+.checkout {
+  padding-left: 21%;
+}
+
+.carticon {
+width: 1.5em;
+margin-right: 1em;
+}
+
+.checkout-button {
+  background-color: #81B642;
+  height: 30px;
+  width: 140px;
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  font-size: 75%;
+  line-height: 30px;
+  border: none;
+  /*do a search on Mozilla Developer's network on line-height. I'm a fan of the multiplier method for these; could =/* Unitless values: use this number multiplied
+by the element's font size
+line-height: 3.5;*/
+  text-align: center;
+  margin: 10px;
+}  
+
+h1 {
+/*Size */
+}
+
+.subbanner {
+  background-color: #ebebeb;
+  height: 4em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  padding-left: 21%;
+}
+
+.subtitle {
+  color: #093b60;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+.message {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 1rem;
+  align-items: center;
+}
+
+.message-image{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.hugpicture {
+  width: 72%;
+  height: 72%;
+}
+
+.blog-preview{
+  display: flex;
+  flex-direction: column;
+  width: 20%;
+}
+
+.miniblog {
+  height: 20%;
+  width: 250px;
+}
+
+.mini-blog-pic{
+  height: 25%;
+}
+
+.mini-blog-text{
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+  color: #3C3C3C;
+}
+
+.mini-blog-link{
+  color: #58b5c9;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+}
+
+h2 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 2em;
+  font-weight: 900;
+}
+
+.here{
+  display: flex;
+  padding-left: 21%;
+}
+
+.lead-in {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: #3C3C3C;
+  width: 72%;
+}
+
+.center {
+  width: 65%;
+}
+
+.profiles {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+
+}
+
+.profile-card {
+  width: 200px;
+  /*You'll want to use a percent for this. You can constrain how small or large it gets with max-width and min-width*/
+  /*I'm probably missing something, but I disagree. I rewrote it with %s and it gets weird when sized down. For these cards, I'm finding px most effective.*/
```

Comment:
As log as it works, It's ok, but I'd argue that setting the size leaves some weird gaps on mobile, and you could scale it up a bit for larger desktops. It would look better fulling more of the screen, and when you're down to one column the tile looks a little small.